lib/active_merchant/billing/gateways/micropayment.rb

```diff
@@ -103,9 +107,9 @@ def add_reference(post, authorization)
       end
 
       def commit(action, params)
-
         params[:testMode] = 1 if test?
         params[:accessKey] = @options[:access_key]
+        params[:apiKey] = @options[:api_key] || @am_api_key
```

Comment:
I'm thinking this could be simplified to:

```
params[:apiKey] = @options[:api_key] || "af1fd841af792f4c50131414ff76e004"
```

And then the creation of the `@am_api_key` on line 18 won't be needed.